README.md

```diff
@@ -58,6 +56,98 @@ timestamps {
 }
 ```
 
+Enable Globally
+=======
+
+You can enable this plugin globally in the Jenkins system configuration page, 
+or with the [configuration as code](https://plugins.jenkins.io/configuration-as-code/) plugin: 
+
+```yaml
+unclassified:
+  logstashConfiguration:
+    enableGlobally: true
+    enabled: true
+    logstashIndexer:
+        logstash:
+            host: "localhost"
+            port: 9200
+```
+
+JobProperty
+=======
+
+This component streams individual log lines to the indexer for post-processing, 
+along with any build data that is available at the start (some information such as the build status is unavailable or incomplete).
+
+Post-Build Publisher
+=======
+
+This component pushes the tail of the job's log to the indexer for post-processing, 
+along with all build data at the time the post-build action had started (if any post-build actions are scheduled after this plugin they will not be recorded).
+
```

Comment:
Can you add the examples of pipeline configuration and the note about sleep workaround from https://plugins.jenkins.io/logstash/#LogstashPlugin-Pipeline too?